susemanager-utils/testing/docker/README.md

```diff
@@ -7,55 +7,46 @@ This is the hierarchy of our containers:
 
 ```
     +----------------+
-    |  sle-<release> |
+    |  opensuse/leap |
     +--------+-------+
              |
              |
              |
-  +----------+---------+
-  | suma-<branch>-base |
-  +----------+---------+
+  +----------+----------+
+  | uyuni-<branch>-base |
+  +----------+----------+
              |
-     +-------+--------------------+
-     |                            |
-     |                            |
-+---------------------+ +---------+---------+
-| suma-<branch>-pgsql | | suma-<branch>-ora |
-+---------------------+ +-------------------+
+             |
+             |
+             |
++----------------------+
+| uyuni-<branch>-pgsql |
++----------------------+
 ```
 
-We have one SUMa specific base container per each branch (1.7, 2.1, ..., head).
-This container is based on "official" SLE container for the SUSE SLE release
-user by the tested branch. These containers are external to the SUMa project,
+We have one Uyuni specific base container per each branch (head).
+This container is based on "official" opensuse container for the release
+used by the tested branch. These containers are external to the Uyuni project,
 they are built with kiwi and we do not have to maintain them.
 
-All the SUMa-related containers are built using
+All the Uyuni-related containers are built using
 [docker's build feature](http://docs.docker.io/en/latest/use/builder/).
 
-From the `suma-<branch>-base` container we create two more containers:
+From the `uyuni-<branch>-base` container we create one additional containers:
 
-  * `suma-<branch>-pgsql`
-  * `suma-<branch>-ora`
+  * `uyuni-<branch>-pgsql`
 
-These containers have all the packages required to test both the Java and the
-Python codebase against either PostgreSQL or Oracle.
+This containers has all the packages required to test both the Java and the
+Python codebase against PostgreSQL.
 
 ## Building
 
 Right now the base containers are build manually, just enter their directory and
 run:
 ```
-docker build -t suma-<branch>-<base|pgsql|ora> .
+docker build -t suma-<branch>-<base|pgsql> .
```

Comment:
I agree with the removal of Oracle, but is this 100% confirmed? I think interlock is still nor closed.

Reply:
Uyuni will never have oracle. So removing it for Uyuni is correct. We will have other containers for SUSE Manager. They may still contain it.

Reply:
Well, maybe Uyuni will not have it, but until it's not 100% confirmed that the removal from SUSE Manager was approved for 4.0, I'd keep the stuff.

Because if for any unlikely reason we stay with Oracle for 4.0, we'll need all the stuff at the Uyuni repository to be able to test internally.

As an alternative, we could wait a little for the PR to be merged, so interlock is ready.